plugins/inputs/tail/tail_test.go

```diff
@@ -9,8 +9,7 @@ import (
 	"time"
 
 	"github.com/google/go-cmp/cmp"
-	"github.com/stretchr/testify/require"
```

Comment:
This import should stay here as we group into _builtin_ packages, _3rd party_ and telegraf _internal_ ones.